tests/unit/modules/election-lookup.js

```diff
@@ -108,7 +108,7 @@ describe('election lookup', function() {
     var titles = $rendered.map(function(idx, elm) {
       return $(elm).find('h3').text().trim();
     }).get();
-    expect(titles).to.deep.equal(['US President', 'NJ Senate', 'NJ House District 09']);
+    expect(titles).to.deep.equal(['US president', 'NJ Senate', 'NJ House District 09']);
```

Comment:
This is a test, not the actual code. Do we not want "president" capitalized in the results here?

![image](https://cloud.githubusercontent.com/assets/1696495/11012274/f24fe8a4-84a6-11e5-90df-26e08008a72d.png)